docs/zkapps/tutorials/03-deploying-to-a-network.mdx

```diff
@@ -190,20 +182,12 @@ A deploy alias consists of:
     NOTE: the private key will be stored in plain text on this computer.
     ```
 
-    A fee payer account is a developer account that can always pay fees immediately for local testing. Do not use an account that holds a substantial amount of MINA.
-
-1. When prompted to create an alias for this account, give an alias to your new fee payer key pair:
-
-    ```text
-    testing-feepayer
-    ```
-
-    Your key pairs and deploy alias are created.
+    Please **do not** use the fee payer account that holds a substantial amount of MINA.
```

Comment:
Good callout here! It might be helpful to highlight why a user should not use a fee payer account that holds a specific amount of mina. Ie a similar message to what we present in the CLI as a warning.

Reply:
In CLI we are printing pretty much the same message:
https://github.com/o1-labs/zkapp-cli/blob/80c2c3a34b6308f5a4ca27b6bb6255005e88662a/src/lib/prompts.js#L216-L217
I'd add the reason to that message.